scripts/build.py

```diff
@@ -171,7 +175,15 @@
     help='Build webpack with source maps.')
 
 
-def generate_app_yaml(deploy_mode=False):
+class DependencyBundleDict(TypedDict):
+    """Dictionary that represents dependency_bundle."""
```

Comment:
```suggestion
    """Dictionary that represents dependency bundle."""
```

Try not to reference variables in docstrings.

Reply:
Okay, I will keep that in mind.